core/modules/system/system.module

```diff
@@ -1294,6 +1303,15 @@ function system_library_info() {
     ),
   );
 
+  // Dismiss buttons.
+  $libraries['backdrop.dismiss'] = array(
```

Comment:
Do we need a library though? This library is going to be just used in one place ever. Could just be a `backdrop_add_js()`. Not a problem TBH, was just curious.

Reply:
Yeah, the javascript code is specific to messages, so it's not like we could use this library anywhere else. I think I'd prefer it wasn't a library.

Reply:
My preference is opposite, I think using a library is always preferable so it can be a dependency or depend on other things. And if we need a CSS file later we have an easy place to add it.